examples/chart/teleport-daemonset/README.md

```diff
@@ -0,0 +1,112 @@
+# Teleport
+
+[Gravitational Teleport](https://github.com/gravitational/teleport) is a modern SSH/Kubernetes API proxy server for
+remotely accessing clusters of Linux containers and servers via SSH, HTTPS, or Kubernetes API.
+
+## Introduction
+
+This chart deploys Teleport components to your cluster via a Kubernetes `Deployment`.
+
+By default this chart is configured as follows:
+
+- 1 replica
+- Record ssh/k8s exec and attach session to the `emptyDir` of the Teleport pod
+  - These sessions will also be stored on the root cluster, if used to access this Helm-configured cluster remotely
+- TLS is enabled by default on the Proxy
+  - The leaf cluster will generate its own self-signed certificates
+
+See the comments in the default `values.yaml` and also the Teleport documentation for more options.
+
+### Setting up your trusted cluster configuration
+
+This version of the chart has been modified to automatically connect back to a "root" Teleport cluster when started. This
+enables remote access to and management of this Teleport cluster (the "leaf" cluster) when deployed on a customer site.
+
+You will need to edit the values in the `trustedCluster.extraVars` section of `values.yaml` as appropriate for your cluster.
+There are comments in the file describing what the values need to be set to.
+
+### DaemonSet information
+
+This version of the chart has also been modified to deploy a Kubernetes `DaemonSet` which will run a privileged pod with a large
+number of host filesystem mounts on every Kubernetes worker node. Each pod will install and set up Teleport to run on the worker
+node itself as a `systemd` service, so that `ssh`-like access to these nodes is possible by logging into the leaf cluster.
+
+The configuration for this is set up in the `daemonset` section of `values.yaml`. You must configure a secure node join token here.
+
+Each node connects back to the auth server of the Teleport leaf cluster running inside Kubernetes by using an exposed `NodePort`.
+This setup may not survive failures of the `kubelet` or other underlying node services, so it should **not** be relied on for
+emergency 'break-glass' access in the event of a failure.
+
+## Prerequisites
+
+- Helm v2.16 (Helm v3 is not currently supported)
+  - You must have Tiller working and configured properly inside your cluster
+- Kubernetes 1.10+
```

Comment:
@webvictim per your status email, do we want to make this 1.15? Since K8s supports the 3 most recent versions with a N-2 support policy?

Reply:
Yeah - I think the chart will actually run on much older versions but I'll update it anyway. Some cloud providers are still using 1.14 as a default and are in the process of updating, so I'll make this 1.14+ instead.